### DISCUSSION TITLE
Same css variables imported multiple times slow down Chrome Inspector

### DISCUSSION THREAD

**Discussion by gvocale** [tailwindlabs/tailwindcss#9877] [Help] [answered]
I am using Tailwind + Next 13 with the /app folder.

`mode: "jit"` is turned on.

I am noticing when running `yarn dev` in Next js that many styles are imported multiple times. This slow down the Chrome inspector sensibly.

Right now I am importing the following:
```tsx
// app/layout.tsx
import "tailwindcss/tailwind.css";
import "../styles/globals.css";
```

```css
// styles/global.css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

See in the attached screenshot how the same css variables are imported multiple times via:
- styles_globals_css.css?ts=1668808838562
- node_modules_pnpm_tailwindcss_3_2_4_postcss_8_4_19_node_modules_tailwindcss_tailwind_css.css?ts=166…
- styles_globals_css.css?ts=1668808823476
... and so on.

Has anyone run into this issue before?

<img width="199" alt="Screenshot 2022-11-18 at 5 15 09 PM" src="https://user-images.githubusercontent.com/110624927/202812201-abbc0969-9750-4ce5-9bf5-b5ba5db2f5f0.png">

<img width="193" alt="Screenshot 2022-11-18 at 5 15 04 PM" src="https://user-images.githubusercontent.com/110624927/202812205-ba7cd7cb-d85e-4afb-80e6-6d03582dc99c.png">

<img width="199" alt="Screenshot 2022-11-18 at 5 14 59 PM" src="https://user-images.githubusercontent.com/110624927/202812211-cedc3eda-26ec-4861-88e6-173a08075199.png">

**Comment by wongjn** [comment, marked as answer]
You're effectively duplicating the Tailwind CSS twice. `tailwindcss/tailwind.css` already contains what you have in `styles/global.css`,

https://github.com/tailwindlabs/tailwindcss/blob/03e3852b6fd1c401c6a32488e118c4670e6e57f0/tailwind.css#L1-L5

so you only need to import **one of** `tailwindcss/tailwind.css` or your own **`styles/global.css`**, but not both.

However, I'm not sure why you'd by getting many loading of the same stylesheet through though!

**Reply by gvocale** [reply]
Great! That fixed it! Thank you so much!

**Reply by gvocale** [reply]
Umh actually that partially fixed it... I still see sometimes multiple classes being imported.

<img width="184" alt="Screenshot 2022-11-18 at 6 00 02 PM" src="https://user-images.githubusercontent.com/1319291/202817113-2cf0f921-9f8c-471c-8b8d-9841418657e1.png">
